pnpm-workspace.yaml

```diff
@@ -2,3 +2,4 @@ packages:
   - "apps/**"
   - "packages/**"
   - "internal/**"
+  - "api"
```

Comment:
Added a `package.json` here. We need pnpm to find it so that the root directory `pnpm-lock.yaml` is built properly.